syntax/index.html

```diff
@@ -349,6 +349,33 @@
       </p>
       <section>
+        <h2>Creating Annotated Tables</h2>
+        <p>After locating metadata, all metadata documents discovered are <a href="http://w3c.github.io/csvw/metadata/#dfn-merged" class="externalDFN">merged</a> into a single description containing all <a title="annotation">annotations</a>. When starting with a metadata file, this involves merging any metadata located about the referenced tabular data files into that metadata file. When starting with a tabular data file this involves locating all metadata as described in <a href="#locating-metadata" class="sectionRef"></a> and merging all relevant metadata into a single descriptor.
+        </p>
+        <p>If processing starts with a tabular data file, implementations:</p>
+        <ol>
+          <li>Retrieve the tabular data file.</li>
+          <li>Retrieve the metadata files described in <a href="#locating-metadata" class="sectionRef"></a>:
+            <ul>
+              <li>metadata supplied by the user (<code>UM</code>) (see <a href="#overriding-metadata" class="sectionRef"></a>)</li>
+              <li>metadata referenced from a <em>Link Header</em> that may be returned when retrieving the tabular data file (<code>LM</code>) (see <a href="#link-header" class="sectionRef"></a>)</li>
+              <li>file-specific metadata in a document located based on the location of the tabular data file (<code>FM</code>) (see <a href="#standard-file-metadata" class="sectionRef"></a>)</li>
+              <li>directory-specific metadata in a document located based on the location of the tabular data file  (<code>DM</code>) (see <a href="#standard-directory-metadata" class="sectionRef"></a>)</li>
+            </ul>
+          </li>
+          <li>Extract the <em>first available</em> <a href="http://www.w3.org/TR/tabular-metadata/#dialect-descriptions">dialect description</a> (<code>DD</code>) from the metadata files <code>UM</code>, <code>LM</code>, <code>FM</code>, <code>DM</code> (in this order) for the table description associated with the tabular data file. If there is no such dialect description, extract the first available dialect description from a table group description in which the tabular data file is described. Otherwise use the default dialect description.</li>
+          <li>If the tabular data file was retrieved with <code>Content-Type</code> including the <code>header=absent</code> parameter set <code>header</code> to <code>false</code> in <code>DD</code>.</li>
+          <li>Parse the tabular data file using <code>DD</code> to create a basic tabular data model (<code>T</code>) with core annotations. Extract embedded metadata (<code>EM</code>), for example from the <a>header line</a>. <span class="note">Note: This specification provides a non-normative definition of how embedded metadata may be extracted from tabular data files in <a href="#parsing-tabular-data" class="sectionRef"></a>.</span></li>
```

Comment:
The note on embedded metadata may be misunderstood as if there was a more explicit informative definition on extracting embedded metadata in the parsing section, though there is not any (which is the way it should be). What about something like

[[[
Note: This specification provides non-normative definition on parsing tabular data which may also include the extraction of embedded metadata in <a href="#parsing-tabular-data" class="sectionRef"></a>. However, no further definition for the format and syntax on embedded metadata is provided. 
]]]

or something like that.